### ISSUE TITLE
Raw tag has no output

### ISSUE THREAD

**Issue by willnationsdev** [sebastienros/fluid#300]
I had an [Issue in OrchardCore](https://github.com/OrchardCMS/OrchardCore/issues/9065) that directed me to open an issue here. The `{% raw %}` and `{% endraw %}` tags no longer produce any output at all, so there is no built-in way to output curly braces and the like. This has become an issue with some of our frontend templates that use JavaScript Handlebars.

While a solution is being worked on, are there any recommendations for temporary workarounds?

**Comment by sebastienros** [comment]
I can't think of any workaround ... working on the solution now

**Comment by willnationsdev** [comment]
@sebastienros As a workaround, we just did line-by-line replacements of raw filters (for now anyway). lol

**Comment by sebastienros** [comment]
So you are generating Handlebars template using Liquid for your frontend?
Have you thought about also using Liquid on the frontend with LiquidJs?

**Comment by willnationsdev** [comment]
@sebastienros We usually don't use Handlebars, but for this one app, we just had to quickly port over an existing app that was already using Handlebars. Almost none of the frontend code has been changed at all, so we were able to get it done quickly. Most of the time, we would likely have done everything with backend code/templates and DisplayAsync. Or if there were client-side templates, they would be Vue, Svelte, or something similar.
